packages/web/src/components/Header/PublicHeader.tsx

```diff
@@ -172,9 +136,7 @@ export const MainHeader: React.FC<MainHeaderProps> = ({
 
   const SearchButton = () => (
     <Button
-      className={classnames(css.searchButton, {
-        [css.searchButtonHide]: minSideNavigation,
-      })}
+      className={classnames(css.searchButton)}
```

Comment:
```suggestion
      className={css.searchButton}
```
Really nitpicking, here